WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -457,6 +457,45 @@ public async Task SignTransactionAsync(TransactionSignaturesRequest request)
 			}
 		}
 
+		public async Task<ReissuanceResponse> ReissuanceAsync(ReissuanceRequest request)
```

Comment:
Ummm this is interesing... I thought this was the same as RegisterOutput because in my mind we have to request the reissuance of amount and vsize but your code made me think that that could not be the case?.... idk. @nothingmuch what do you say?

Reply:
Di we have to break down the vsize credentials.

Reply:
Added vsize cred reissuance.

Reply:
we probably don't need to split/consolidate vsize ones too often since there will likely be a surplus value but its bound to come up at least occasionally, and we definitely will need the same number of credentials overall, including zero credentials, so yes we need to do both kinds IMO

Reply:
I guess this is done.

Reply:
still needs zero vsize credentials